ani-cli

```diff
@@ -323,7 +323,10 @@ while [ $# -gt 0 ]; do
         -s | --syncplay)
             case "$(uname -s)" in
                 Darwin*) player_function="/Applications/Syncplay.app/Contents/MacOS/syncplay" ;;
-                MINGW* | *Msys) player_function="/c/Program Files (x86)/Syncplay/Syncplay.exe" ;;
+                MINGW* | *Msys)
+                    player_function="/c/Program Files (x86)/Syncplay/Syncplay.exe"
+                    command -v "$player_function" >/dev/null || player_function="syncplay.exe"
+                    ;;
```

Comment:
I like the approach, however I'd do it in reverse: first check if it's in the path then program files (x86).

Reply:
I thought of that, but one thing to note about this approach the error message from `dep_ch()` will give will simply be `Program "syncplay.exe" not found. Please install it.` So, a user would install Syncplay either through the normal installer / choco or through scoop.

If I reverse it, the error message from `dep_ch()` will be `Program "/c/Program Files (x86)/Syncplay/Syncplay.exe" not found. Please install it.` This will be inconsistent with the other error messages. For example, if fzf is not found the error message will be `Program "fzf" not found. Please install it.`

Another thing to note is that the "default" location for Syncplay may be in `/c/Program Files (x86)`.

Would you still like me to reverse it?

Reply:
Oh, that makes sense... Another question is why is it not a good idea to add `Program Files (x86)` to path (because if we can get away with just that, then I'd rather have it that way...)

Reply:
It could simply be:
```
MINGW* | *Msys) player_function="syncplay.exe" ;;
```
But Windows users would have to manually add it to the path if they downloaded it through means other than the scoop package manager. Then we might have issues like "Why can't ani-cli detect Syncplay"?
We could go with that since we're already asking them to do a lot of setup (like getting a new terminal and adding git bash as a profile it).

Reply:
Or do you mean something like this:
```
MINGW* | *Msys)
    export PATH=$PATH:"/c/Program Files (x86)/Syncplay/"
    player_function="syncplay.exe"
    ;;
```
This should cover for both download locations. It works on my machine despite Syncplay.exe being capitalized in `/c/Program Files (x86)/Syncplay/`.